src/System.Private.Xml/tests/XmlReader/XmlResolver/XmlSystemPathResolverTests.cs

```diff
@@ -90,10 +90,13 @@ public static void TestResolveInvalidPaths()
             AssertInvalidPath("??");
         }
 
-        [Fact]
-        public static void TestResolveInvalidPath()
+        [Theory]
+        [InlineData("http://F195497AB14A4DFB94122F9B05EBF4A1")]
```

Comment:
why is this an invalid path?  if the host name "F195497AB14A4DFB94122F9B05EBF4A1" exists, then it would work fine. In fact, with some ISPs, that hostname would resolve to some page where the ISP presents a special page for "host not found". This is similar to what @wfurt changed in the sockets tests. Perhaps this could be changed to something like "http://notfound.invalid.corp.microsoft.com" or something like that.